integration-cli/docker_cli_search_test.go

```diff
@@ -11,83 +12,50 @@ func (s *DockerSuite) TestSearchOnCentralRegistry(c *check.C) {
 	testRequires(c, Network, DaemonIsLinux)
 
 	out, exitCode := dockerCmd(c, "search", "busybox")
-	if exitCode != 0 {
-		c.Fatalf("failed to search on the central registry: %s", out)
-	}
+	c.Assert(exitCode, check.Equals, 0, check.Commentf("failed to search on the central registry: %s", out))
```

Comment:
It's not losing anything since exitCode is always 0 and this will never be displayed.
Also, it's the name of the test and fairly easy to get at.